src/com/yetanalytics/lrs_admin_ui/views/footer.cljs

```diff
@@ -1,8 +1,21 @@
-(ns com.yetanalytics.lrs-admin-ui.views.footer)
+(ns com.yetanalytics.lrs-admin-ui.views.footer
+  (:require [goog.string :refer [format unescapeEntities]]
+            goog.string.format))
 
 (defn footer []
   [:footer {:class "home-footer"}
    [:div {:class "footer-wrapper"}
-    [:span "@Copyright 2021"]"   |  "
-    [:a {:class "text-white", :href "#"} "Terms and Conditions"]"   |  "
-    [:a {:class "text-white", :href "#"} "Privacy Policy"]]])
+    [:span {:class "text-white"}
+     (format "%s2021 " (unescapeEntities "&copy;"))
+     [:a {:href "https://www.yetanalytics.com"
+          :target "_blank"
+          :class "text-white"}
+      "Yet Analytics Inc."]]"   |   "
+    [:a {:class "text-white", :href "https://github.com/yetanalytics/lrsql/blob/main/LICENSE"}
+     "Licensed under the Apache 2.0 License"]"   |   "
+    [:a {:class "text-white", :href "https://github.com/yetanalytics/lrsql"}
+     "Contribute on "
+     [:img {:src "images/icons/github.png"
+            :alt "Github Logo"}]]
+    [:span {:class "support-note"}
+     "Contact us to learn about Enterprise Support options."]]])
```

Comment:
This could be a place to add a mailto link

Reply:
its going to a contact form on the website. Form does not exist yet, Shelly is making. Will update when it does.

Reply:
So will adding the contact form link be part of this PR or a future one?

Reply:
future.